app/controllers/profile/base_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Profile::BaseController < ApplicationController
+
+end
```

Comment:
What is the role of this controller? Is it going to be needed moving forward?

Reply:
We actually *don't* need it now - good catch. I can delete this and we can add it if we find a use case.